src/main/com/fulcrologic/fulcro/routing/dynamic_routing.cljc

```diff
@@ -314,18 +314,38 @@
         root       (ast-node-for-live-router app ast)
         to-signal  (atom [])
         to-cancel  (atom [])
-        _          (loop [{:keys [component] :as node} root new-path-remaining new-path]
+;; TODO: Remove next line
+;;        _          (loop [{:keys [component] :as node} root new-path-remaining new-path]
+        _          (loop [{:keys [component children] :as node} root new-path-remaining new-path]
                      (when (and component (router? component))
                        (let [new-target    (first new-path-remaining)
                              router-ident  (comp/get-ident component {})
                              active-target (get-in state-map (conj router-ident ::current-route))
                              {:keys [target]} (get-in state-map (conj router-ident ::pending-route))
-                             next-router   (some #(ast-node-for-live-router app %) (:children node))]
```

Comment:
You can see here that app is exactly how I ported the old version

Reply:
Yup, noted :+1: